src/SQLStore/QueryDependency/DependencyLinksValidator.php

```diff
@@ -169,7 +169,7 @@ public function hasArchaicDependencies( DIWikiPage $subject ) {
 		$links_table = $connection->tableName( SQLStore::QUERY_LINKS_TABLE );
 
 		$row = $connection->selectRow(
```

Comment:
Same comment for the join_conditions line 183 and the now-unused variable `$links_table`.